controls/radlayoutcontrol/features/toolbox/layoutcontroltoolboxview.md

```diff
@@ -0,0 +1,63 @@
+---
+title: LayoutControlToolBoxView
+page_title: LayoutControlToolBoxView
+description: LayoutControlToolBoxView
+slug: radlayoutcontrol-features-toolbox-toolboxview
+tags: layout, toolbox, layoutcontroltoolbox, layoutcontroltoolboxview
+published: True
+position: 1
+---
+
+# LayoutControlToolBoxView
+
+__LayoutControlToolBoxView__ is the control that represents the toolbox. This article demonstrates how to set it up and associate it with RadLayoutControl. 
+
+#### __[XAML] Example 1: Setting up the toolbox__  
+	 <StackPanel Orientation="Horizontal">
+        <telerik:RadLayoutControl x:Name="layout" Width="300" Height="250">
+            <Button Content="Item" />
+            <telerik:LayoutControlGroup Orientation="Vertical">
+                <Button Content="Item" />
+                <Button Content="Item"/>
+            </telerik:LayoutControlGroup>
+        </telerik:RadLayoutControl>
+        <telerik:LayoutControlToolBoxView LayoutControl="{Binding ElementName=layout}" Width="300" Height="250"/>
+    </StackPanel>
+	
+The toolbox can be associated with RadLayoutControl through its __LayoutControl__ property (see __Example 1__).
+	
+#### __Figure 1: Final result__  
+![](images/radlayoutcontrol-features-toolbox-toolboxview-01.png)
+
+>Drop from the toolbox is allowed only if the layout control is in [edit mode]({%slug radlayoutcontrol-edit-the-layout%}).
+
+You can __get, add or remove__ the elements from the toolbox using the __NewItems__ and __DeletedItems__ collections of the __LayoutControlToolBoxView__ control.
+
+## Tracking the deleted items
+
+The toolbox allows you to track and restore the items that was removed from the layout. They are displayed in the __Deleted Items__ visual section of the control. Once an item is deleted, it is added in the __DeletedItems__ collection of the LayoutControlToolBoxView. You can restore the element back in the layout by drag/droping it from the Deleted Items to the __RadLayoutControl__.
```

Comment:
The toolbox allows you to track and restore the items that were removed from the layout. [...] Once an item is deleted, it is added to the __DeletedItems__ collection of the LayoutControlToolBoxView. You can restore the element back in the layout by drag/dropping it from the Deleted Items to the __RadLayoutControl__.